profiler_measurements/utils.h

```diff
@@ -0,0 +1,40 @@
+
+#ifndef TESTLIBRARY_UTILS_H
+#define TESTLIBRARY_UTILS_H
+
+#include <termios.h>
```

Comment:
These libraries will not compile on many platforms. Can't we use something els which is cross-platfrom? Do we need keyboard stuff for testing?